packages/experimental/RadioGroup/src/Radio.types.ts

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IFocusable } from '@fluentui-react-native/interactive-hooks';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+
+export const radioName = 'Radio';
+
+export interface RadioProps extends IPressableProps {
+  /**
+   * The text string for the option
+   */
+  label: string;
+
+  /**
+   * A unique key-identifier for each option
+   */
+  value: string;
+
+  /**
+   * Whether or not the radio button is selectable
+   */
+  disabled?: boolean;
+
+  /**
+   * An optional string for the Narrator to read for each RadioButton. If not provided, this will be set to the button's content.
+   */
+  accessibilityLabel?: string;
+
+  /**
+   * Defines the current radio button's position in the radio group for accessibility purposes. It's recommended to set this value
+   * if radio buttons are not direct children of radio group. This value is auto-generated if radio buttons are direct children of
+   * radio group.
+   */
+  accessibilityPositionInSet?: number;
+
+  /**
+   * Defines the number of radio buttons in the group for accessibility purposes.It's recommended to set this value if radio
+   * buttons are not direct children of radio group. This value is auto-generated if radio buttons are direct children of
+   * radio group.
+   */
+  accessibilitySetSize?: number;
```

Comment:
I don't think you need to define `accessibilityLabel`, `accessibilityPositionInSet`, and `accessibilitySetSize` props here. They should be included as part of View props which you can set on the root.